src/System.Net.Security/src/System/Net/SslStreamPal.Unix.cs

```diff
@@ -75,19 +75,9 @@ public static void QueryContextStreamSizes(SafeDeleteContext securityContext, ou
             streamSizes = new StreamSizes();
         }
 
-        public static int QueryContextConnectionInfo(SafeDeleteContext securityContext, out SslConnectionInfo connectionInfo)
+        public static void QueryContextConnectionInfo(SafeDeleteContext securityContext, out SslConnectionInfo connectionInfo)
```

Comment:
If we're no longer returning an int, any reason not to change the out parameter to be the return value?